src/Algolia.Search/Models/Common/LogResponse.cs

```diff
@@ -37,6 +37,36 @@ public class LogResponse
         public IEnumerable<Log> Logs { get; set; }
     }
 
+    /// <summary>
+    /// https://www.algolia.com/doc/api-reference/api-methods/get-logs/#method-response-inner_queries
+    /// Contains an object for each performed query with the indexName, queryID, offset, and userToken.
+    /// </summary>
+    public class InnerQueryLog
+    {
+        /// <summary>
+        /// The index used for the given query
+        /// </summary>
+        [JsonProperty(PropertyName = "index_name")]
+        public string IndexName { get; set; }
+
+        /// <summary>
+        /// The QueryID for the given query
+        /// </summary>
+        [JsonProperty(PropertyName = "query_id")]
+        public string QueryId { get; set; }
```

Comment:
From your example, shouldn't the `QueryId` be a `long?`?

Reply:
Actually, it's a string (I double checked in the engine code base). My example is wrong I am going to fix it.